src/yarpdataplayer-gtk/include/msvc/dirent.h

```diff
@@ -0,0 +1,372 @@
+/*****************************************************************************
```

Comment:
Can this be rationalized with existing dirent stuff? E.g. ACE has dirent (and ACE is required to compile yarp with MSVC), other platforms have it natively, and @apaikan also has a copy of the Ronkko implementation at `src/libYARP_manager/include/yarp/manager/ymm-dir.h`